src/dune/cinaps.ml

```diff
@@ -110,4 +111,10 @@ let gen_rules sctx t ~dir ~scope =
          :: List.map cinapsed_files ~f:(fun fn ->
                 let fn = Path.build fn in
                 A.diff ~optional:true fn
-                  (Path.extend_basename fn ~suffix:".cinaps-corrected")) )))
+                  (Path.extend_basename fn ~suffix:".cinaps-corrected")) ))
+  in
+  Super_context.add_alias_action sctx ~dir ~loc:(Some loc) ~stamp:"cinaps"
+    (alias ~dir) action;
+  Super_context.add_alias_action sctx ~dir ~loc:(Some loc) ~stamp:"cinaps"
+    (Alias.runtest ~dir)
+    (Build.alias (alias ~dir) >>> Build.noop)
```

Comment:
This is a heavy way to add a dependency to an alias. In particular, adding an action to an alias is more expensive than adding a plan dependency. So instead, we should add a dependency on `Alias.stamp_file (alias ~dir)`.

We should also bind `alias ~dir` to a variable given that it's used several times in this function.

Reply:
That's good to know, I wasn't familiar enough with these parts. Just fixed it!

Sorry for the delay

Reply:
No problem :)